src/style.css

```diff
@@ -0,0 +1,36 @@
+* {
+  margin: 0;
+}
+
+.box {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  width: 96px;
+  height: 96px;
+}
+
+.stars {
+  display: flex;
+  justify-content: space-between;
+}
+
+.stars__star {
+  background-image: url(images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+  width: 16px;
+  height: 16px;
+}
+
+.stars--1 :first-child,
+.stars--2 > :nth-child(-n + 2),
+.stars--3 > :nth-child(-n + 3),
+.stars--4 > :not(:last-child),
+.stars--5 > :nth-child(n) {
+  background-image: url(images/star-active.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+  width: 16px;
+  height: 16px;
```

Comment:
These styles are same as in `.stars__star`, so there is no need to overwrite them. Only image changes.